lib/active_fedora/core.rb

```diff
@@ -139,7 +139,9 @@ def id_to_uri(id)
           translate_id_to_uri.call(id)
         else
           id = "/#{id}" unless id.start_with? '/'
-          id = ActiveFedora.fedora.base_path + id unless id.start_with? "#{ActiveFedora.fedora.base_path}/"
+          unless ActiveFedora.fedora.base_path == '/'
```

Comment:
Would you please combine the two conditionals: 
`unless ActiveFedora.fedora.base_path == '/' || id.start_with? "#{ActiveFedora.fedora.base_path}/"`

also freeze the string to prevent allocation on each trip through here: `'/'.freeze`